nidaqmx/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import nidaqmx
+from nidaqmx.grpc_session_options import GrpcSessionOptions
+import grpc
+
+@pytest.fixture(scope="module")
+def grpc_session_creation_kwargs():
+        channel = grpc.insecure_channel(f"localhost:31763")
+        grpc_options = GrpcSessionOptions(
+            grpc_channel = channel,
+            session_name = "NIDAQmxSession",
```

Comment:
Some tests create multiple tasks. By specifying a hardcoded session name, they will end up sharing a task on the grpc-device server, which is incorrect.

nimi-python's tests leave the session name blank, so the grpc-device server will assign each task a unique session name. I think we should do that here.